config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/raffaelespazzoli/cert-utils-operator
```

Comment:
shouldnt it be `quay.io/redhat-cop/cert-utils-operator

Reply:
that field is all over the place in the other operators, probably it does not matter. anyway I fixed it

Reply:
agree. only reason I called this one out was that it was not a valid image path. Thanks for updating.